packages/fxa-content-server/app/scripts/views/index.js

```diff
@@ -214,12 +222,24 @@ class IndexView extends FormView {
           // the next page.
           account = this.user.getAccountByEmail(email);
           // the returned account could be the default,
-          // ensure it's email is set.
+          // ensure its email is set.
           account.set('email', email);
+          this.navigate(nextEndpoint, { account });
+        } else {
+          // The email address does not belong to a current user. Validate its
+          // domain name.
+          if (this.isInExperimentGroup('emailMxValidation', 'treatment')) {
```

Comment:
For this experiment to be reported correctly, you'll need to do a bit more setup.

First, you'll need to add `emailMxValidation` to the list [lib/experiment.js](https://github.com/chenba/fxa/blob/6e5714881980035daa086bdd2dddd9674f67654b/packages/fxa-content-server/app/scripts/lib/experiment.js#L23).

And here, you need to call [createExperiment](https://github.com/chenba/fxa/blob/6e5714881980035daa086bdd2dddd9674f67654b/packages/fxa-content-server/app/scripts/lib/experiment.js#L170) with the experiment group.

You can do something like 

```js
const group = this.getExperimentGroup('emailMxValidation');
this.createExperiment('emailMxValidation', group);
if (group === 'treatment') {
...
}
```

What will the ultimate measure of success be, whether the user has verified their email?

Reply:
Thanks!

Yep, ultimate measure of success is sign up completion.